person.rb

```diff
@@ -0,0 +1,36 @@
+class Person
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options = {})
+    @age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+  end
+
+  def offer_high_five
+    "High five!"
+  end
+
+  module Phaseable
```

Comment:
whoa why are these modules defined inside the `person` class?  Should be defined in their own files, and included in whichever files they are needed (in this case the student class)